pkg/config/config.go

```diff
@@ -101,6 +101,9 @@ type OS struct {
 	Password       string            `json:"password,omitempty"`
 	Environment    map[string]string `json:"environment,omitempty"`
 	Labels         map[string]string `json:"labels,omitempty"`
+
+	// ForceMBR is not a cOS installation flag so it's not in "Install" struct
+	ForceMBR bool `json:"forceMbr,omitempty"`
```

Comment:
`ForceGPT` is in `.install`. Should we move `ForceMBR` there? Or move `ForceGPT` here.

Reply:
I think we need to keep `ForceGPT` in `.install` because only the configs in `.install` get transformed into env vars (i.e. `COS_INSTALL_FORCE_GPT`) and passed to cOS installer. We need cOS installer to force GPT for BIOS systems in order to make custom partition layout to work.

Reply:
+1 to put ForceMBR and ForceGPT together. It makes more sense from UX perspective regardless of the underlying implementation. Will it cause trouble if we also transform FORCE_MBR to cOS?

Reply:
It's fine for now because cOS-installer has no such config as `COS_INSTALL_FORCE_MBR` so it won't affect cOS installer's behavior. `ForceMBR` was not in the `.install` struct because I don't want to declare an unrelated env. variable that's not used by cOS-installer. But if it makes more sense from the configuration point of view, I think we can move `ForceMBR` to `install`.

Reply:
@gitlawr @bk201 Moved `ForceMBR` to `install` struct.